app/models/manageiq/providers/container_manager.rb

```diff
@@ -12,7 +12,7 @@ class ContainerManager < BaseManager
     has_many :container_limits, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_image_registries, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_images, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :persistent_volumes, :as => :parent, :dependent => :destroy
+    has_many :persistent_volumes, :foreign_key => :parent_id, :dependent => :destroy
```

Comment:
So the idea here is to retrieve the persistent volumes of a container manager without using the `parent_type` attribute because the value that was set there is wrong: "ManageIQ::Providers::ContainerManager" instead of "ExtManagementSystem" which is the base class and thus the correct one to use here. 
As I mentioned in the PR description we have another PR against master which includes a migration that fixes this issue. Since we don't want to backport that migration we will use this solution. It works because checking the type of the object (`PersistentVolume`) without checking the parent_type is enough.